### ISSUE TITLE
Simplify configuration : remove enableOutdoor

### ISSUE THREAD

**Issue by babastienne** [GeotrekCE/Geotrek-rando-v3#977]
IMO if there is some published outdoor contents, it means users wants them displayed in rando, so they shouldn't have to enable the option in the configuration of GTR3.

When the configuration is enable if there is no content the Outdoor filters are not shown anyway.

I believe we don't need the configuration to activate/deactivate Outdoor, don't you think @camillemonchicourt ? Are you okay to remove it in order to simplify the configuration of Geotrek ?

**Comment by camillemonchicourt** [comment]
Oui, au départ on a ajouté le paramètre `enableOutdoor` car le module Outdoor était tout jeune dans Geotrek-admin, mais aussi car il n'était pas installé par défaut.
Et ainsi si il n'était pas installé côté Geotrek-admin et que GTR essayait d'interroger les routes de l'API du module Outdoor, celles-ci n'existaient pas et ça faisait planter GTR.

Je ne sais plus si désormais on installe le module Outdoor par défaut (on pourrait je pense si ce n'est pas le cas), mais l'idéal est en effet de supprimer ce paramètre et d'afficher le module dans GTR si au moins un site Outdoor est renvoyé par l'API.
Si cela n'est pas plus lourd pour construire la page de recherche.

Mais il faut veiller au cas où le module Outdoor n'est pas présent dans Geotrek-admin et donc que les routes API n'existent pas.

**Comment by babastienne** [comment]
Ok merci pour ta confirmation. Je pense en effet qu'on va retirer ce paramètre pour simplifier.

Côté Admin le module Outdoor n'est toujours pas activé par défaut mais ça serait bien qu'il le soit, je vais créer un ticket en ce sens pour que les modules soient tous activés par défaut (outdoor, sensitivity).

**Comment by dtrucs** [comment]
Dans la même idée il y'a également dans la config le booléen `enableTouristicEvents` (définie à `true` par défaut)
